src/index.ts

```diff
@@ -217,6 +219,78 @@ export class Framecast {
     return this.call('evaluate', calledFnString);
   }
 
+  /**
+   * Creates state that is synced between the two windows
+   */
+  state<StateType = any>(
+    key: string,
+    initialValue: StateType
+  ): WritableAtom<StateType> {
+    let isInitialValue = true;
+    const $atom = atom<StateType>(initialValue);
```

Comment:
Should we populate this initial value here instead of just waiting for the `state:get:foo`?

I'm thinking it will cause two updated when there is already a initial value on the parent. It will create the atom with the child initial value and then have an update to the parent initial value.